packages/codemod/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@commercetools-frontend/codemod",
+  "version": "0.0.1",
+  "description": "Codemod transformations for Custom Applications",
+  "bugs": "https://github.com/commercetools/merchant-center-application-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/merchant-center-application-kit.git",
+    "directory": "packages/codemod"
+  },
+  "homepage": "https://docs.commercetools.com/custom-applications",
+  "keywords": ["javascript", "frontend", "codemod", "toolkit"],
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": ["build", "package.json", "LICENSE", "README.md"],
+  "bin": {
+    "mc-codemod": "./bin/cli.js"
+  },
+  "dependencies": {
+    "glob": "8.0.1",
+    "is-git-clean": "1.1.0",
```

Comment:
I am wondering why are we adding this as a dependency?

Reply:
Ah yeah I wanted to implement this check but decided to omit it for now. I'll remove the dependency. Thanks for checking.